packages/sdk-react/src/components/userRoles/UserRoleDetails/UserRoleDetails.tsx

```diff
@@ -20,7 +35,11 @@ export const UserRoleDetails = (props: UserRoleDetailsProps) => (
   </MoniteScopedProviders>
 );
 
-const UserRoleDetailsBase = ({ id }: UserRoleDetailsProps) => {
+const UserRoleDetailsBase = ({
+  id,
+  onUpdated,
+  onCreated,
+}: UserRoleDetailsProps) => {
   const { i18n } = useLingui();
```

Comment:
the `role` variable seems not using here (line 47):
```ts
const {
    isLoading,
    isPending,
    data: role,
    error: roleQueryError,
  } = useRoleById(id);
```

Reply:
fixed